stretch-assignment/digital_timer/index.js

```diff
@@ -0,0 +1,22 @@
+
+
+
+function counter() {
+    let count = 1;
+    setInterval(function() {
+        
+        if (count < 10) {
+            document.querySelector('#secondOnes').textContent = count;
+            document.querySelector('#secondTens').textContent = 0;
+            count += 1;
+        }
+        else if (count === 10){
+            document.querySelector('#secondTens').textContent = 1;
+            document.querySelector('#secondOnes').textContent = 0;
+            count = 0;
+        }
+    }, 1000);
```

Comment:
Nice job just needs milliseconds! but i mean its like the same thing as seconds just a bit quicker on the interval